tracee/consts.go

```diff
@@ -1565,3 +1571,8 @@ var EventsIDToParams = map[int32][]param{
 	MemProtAlertEventID:        {{pType: "alert_t", pName: "alert"}},
 	SchedProcessExitEventID:    {},
 }
+
+const (
+	FilterUIDCeiling = int64(math.MaxUint32 + 1)
+	FilterUIDFloor   = int64(-1)
```

Comment:
If this isn't API you want to export I can maneuver checking of the values into the BPF code.

Reply:
I don't think we need these consts.
But what do you mean maneuver checking the values in the BPF code?

Reply:
I meant that we could check for default values in BPF instead of NULL. In that case the map would be updated regardless of if they were set in Go code. I think this is fine as is, got rid of the consts though.